data/org.freedesktop.PolicyKit1.AuthenticationAgent.xml

```diff
@@ -1,3 +1,28 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!--
+  Copyright 2009, 2015 Red Hat
+
+  SPDX-License-Identifier: LGPL-2.1-or-later
```

Comment:
The SPDX line should be a separate comment, on a single line (see 'Format'): https://spdx.dev/learn/handling-license-info/#how

Reply:
I think that might be an artifact of the fact the guidance on spdx.dev is using single line comment delimiters. The [guidance on reuse.software](https://reuse.software/tutorial/) shows the SPDX lines can be incorporated into the wider header comment, `reuse lint` is happy with this format, and it’s the way I (and others) have been doing it across multiple projects for a while now!

Reply:
Ok fair enough